x-pack/legacy/plugins/uptime/public/pages/settings.tsx

```diff
@@ -32,14 +31,15 @@ import {
   CertificateExpirationForm,
   OnFieldChangeType,
 } from '../components/settings/certificate_form';
+import * as Translations from './translations';
 
 const getFieldErrors = (formFields: DynamicSettings | null) => {
   if (formFields) {
     const blankStr = 'May not be blank';
     const { certificatesThresholds, heartbeatIndices } = formFields;
     const heartbeatIndErr = heartbeatIndices.match(/^\S+$/) ? '' : blankStr;
-    const errorStateErr = certificatesThresholds?.errorState ? null : blankStr;
-    const warningStateErr = certificatesThresholds?.warningState ? null : blankStr;
+    const errorStateErr = certificatesThresholds?.expiration ? null : blankStr;
+    const warningStateErr = certificatesThresholds?.age ? null : blankStr;
```

Comment:
can you also rename var name?

Reply:
I realized we needed to improve this, so I added a type and renamed the var usage in https://github.com/elastic/kibana/pull/64111/commits/59e60e8b5b830f0fc21fd793cd75137f9675124e.